compiler/modules/nback/nback.bal

```diff
@@ -6,11 +6,14 @@ import wso2/nballerina.nback.llvm;
 
 # The preferred output extension for the output filename.
 public const OUTPUT_EXTENSION = ".ll";
+configurable string target = "x86_64-pc-linux-gnu";
 
 // If outputFilename is non-nil write the output to this file.
-public function compileModule(bir:Module mod, string? outputFilename) returns err:Any|io:Error? {
+public function compileModule(bir:Module mod, string? outputFilename) returns err:Any|io:Error|error? {
```

Comment:
This should use a distinct error type. In nback.bal, add


public type ConfigError distinct error;

Reply:
Fixed